pkg/rbenv-ruby-2.3.0/srcurl

```diff
@@ -0,0 +1 @@
+http://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.0.tar.gz
```

Comment:
No, as then it's inconsistent with the other existing files.. I'd be happy to work on a story that requests a change to them all if we decide to go that way.